packages/playground/examples/store/main.ts

```diff
@@ -12,7 +12,9 @@ function initDoc() {
   const workspace = new DocCollection({ schema });
   workspace.meta.initialize();
   const doc = workspace.createDoc({ id: 'doc:home' });
-  doc.addBlock('todo:container');
+  doc.load();
```

Comment:
I think the more semantic pattern should be

```ts
doc.load(() => {
  const rootId = doc.addBlock('todo:root');
  doc.addBlock('todo:container', {}, rootId);
})
```

Reply:
Ok, I create another PR for this. .👍